src/query/management/src/udf/udf_mgr.rs

```diff
@@ -121,16 +121,24 @@ impl UdfMgr {
     /// Get all the UDFs for a tenant.
     #[async_backtrace::framed]
     #[minitrace::trace]
-    pub async fn list_udf(&self) -> Result<Vec<UserDefinedFunction>, UdfApiError> {
```

Comment:
maybe we add another function named `list_udf_fallback` when the error occurs.

Reply:
Ok, it is safer to deal only with error cases. Please review the newest commit.